src/expr.rs

```diff
@@ -348,32 +630,111 @@ pub enum BinaryOperation<'a> {
 
 #[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
 #[serde(rename_all = "lowercase")]
-/// Verdict expression.
+/// A verdict expression (used in [verdict maps](crate::stmt::VerdictMap)).
+///
+/// There are also verdict [statements](crate::stmt::Statement), such as
+/// [accept](crate::stmt::Statement::Accept).
 pub enum Verdict<'a> {
+    /// Terminate ruleset evaluation and accept the packet.
+    ///
+    /// The packet can still be dropped later by another hook, for instance
+    /// accept in the forward hook still allows one to drop the packet later in
+    /// the postrouting hook, or another forward base chain that has a higher
+    /// priority number and is evaluated afterwards in the processing pipeline.
     Accept,
+    /// Terminate ruleset evaluation and drop the packet.
+    ///
+    /// The drop occurs instantly, no further chains or hooks are evaluated.
+    /// It is not possible to accept the packet in a later chain again, as those
+    /// are not evaluated anymore for the packet.
     Drop,
+    /// Continue ruleset evaluation with the next rule.
+    ///
+    /// This is the default behaviour in case a rule issues no verdict.
     Continue,
+    /// Return from the current chain and continue evaluation at the next rule
+    /// in the last chain.
+    ///
+    /// If issued in a base chain, it is equivalent to the base chain policy.
     Return,
+    /// Continue evaluation at the first rule in chain.
+    ///
+    /// The current position in the ruleset is pushed to a call stack and
+    /// evaluation will continue there when the new chain is entirely evaluated
+    /// or a [return](Verdict::Return) verdict is issued. In case an absolute
+    /// verdict is issued by a rule in the chain, ruleset evaluation terminates
+    /// immediately and the specific action is taken.
     Jump(JumpTarget<'a>),
+    /// Similar to jump, but the current position is not pushed to the call
+    /// stack, meaning that after the new chain evaluation will continue at the
+    /// last chain instead of the one containing the goto statement.
```

Comment:
```suggestion
    /// Similar to jump, but the current position is not pushed to the call
    /// stack.
    ///
    /// That means that after the new chain evaluation will continue at the
    /// last chain instead of the one containing the goto statement.
```

Reply:
Done